packages/fast-check/test/unit/arbitrary/_internals/implementations/SchedulerImplem.spec.ts

```diff
@@ -1028,6 +1028,40 @@ describe('SchedulerImplem', () => {
       }
     });
 
+    it('should never overlap tasks from function and sequences', async () => {
```

Comment:
Actually it does not detect our issues but at least it could help us into detecting others.